### DISCUSSION TITLE
(libreoffice-fresh): add the libreoffice sdk as a 'sub-package'

### DISCUSSION THREAD

**Discussion by esohns** [chocolatey-community/chocolatey-packages#2105] [Ideas]
would it be possible to add the libreoffice sdk as a package as well (same as the 'help' package) ? I need it for my project.

**Comment by AdmiringWorm** [comment]
We currently do not accept completely new packages (except if it replaces one we already maintain).

We may consider including it in this repository if you already own the package on Chocolatey Community Repository. Still, we do not generally accept unmaintained packages (which completely new packages are considered as).
Unfortunately, at the moment we have a temporary hold on accepting migration of packages as well at least until we have updated our contribution guidelines.

**Reply by esohns** [reply]
thanks for the update. I have made a package in the meantime: https://community.chocolatey.org/packages/libreoffice-sdk/7.5.0 which is currently in review.
